src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -60,6 +60,131 @@ using var meterProvider = Sdk.CreateMeterProviderBuilder()
 Refer to [Program.cs](../../examples/runtime-instrumentation/Program.cs) for a
 complete demo.
 
+## Metrics
+
+### GC related metrics
+
+| Name                                            | Description                                                          | Units     | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|-------------------------------------------------|----------------------------------------------------------------------|-----------|-------------------|------------|------------------|------------------|
+| process.runtime.dotnet.**gc.collections.count** | Number of times garbage collection has occurred since process start. | `{times}` | ObservableCounter | `Int64`    | gen              | gen0, gen1, gen2 |
```

Comment:
Do we need to specify that we internally use ObservableCounter? Which instrument we use should be implementation detail, so can be omitted.

Reply:
I'm using it because all the semantic convention docs mention the instrument type, e.g. [JVM metrics](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md#jvm-metrics).

Plus, the Prometheus exporter does expose whether it's a counter or a gauge, so I think this information somewhat relevant:

![image](https://user-images.githubusercontent.com/9310587/176552098-f4339ede-c3ba-46a2-98e3-c1dc6944c3d9.png)

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#instrument-types spec notes.

Not a blocker, okay to keep it here.